FpsCheck.py

```diff
@@ -339,7 +339,16 @@ def check_list_sites(self, primary, site_list):
                 self.error_list.append(
                     "Experienced an error when trying to access " + url + "; "
                     + "error was: " + str(inst))
-                
+    
+    def check_well_known_list(self, field, list1, list2):
+        """Checks that 2 lists for a given field match each other"""
```

Comment:
Same something about the return value here? E.g. "Returns a list containing all errors found (as strings)."